src/nft/queries/genie/TrendingTokensFetcher.ts

```diff
@@ -1,6 +1,18 @@
+import { nativeOnChain, WRAPPED_NATIVE_CURRENCY } from 'constants/tokens'
+
 import { FungibleToken } from '../../types'
 
+function replaceWrappedWithNative(token: FungibleToken) {
```

Comment:
i can't quite articulate the way to combine this and the other replace wrapped function so i'm okay leaving them for now

Reply:
They're in wildly different parts of the codebase, otherwise I'd throw it in a shared utils.

Reply:
Moved to src/utils/unwrapToken.ts.